lib/swagger/formatter.ex

```diff
@@ -1,105 +1,336 @@
 defmodule Xcribe.Swagger.Formatter do
-  @moduledoc """
-  Creates Swagger maps according to OpenAPI Specification
+  @moduledoc ~S"""
+  Format a given `Xcribe.Request` according to OpenAPI Specification.
+
+  To know more about the specifications [OpenAPI 3.0.3](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md)
   """
+
   alias Xcribe.{JSON, Request}
+  alias Xcribe.Swagger.Types
+
+  import Xcribe.Helpers.Formatter, only: [content_type: 1, authorization: 1]
+
+  @doc """
+  Return an empty struct of an OpenAPI Object.
+  """
+  def raw_openapi_object do
+    %{
+      openapi: "3.0.3",
+      info: nil,
+      servers: nil,
+      paths: nil,
+      components: nil
+    }
+  end
 
-  import Xcribe.Swagger.Descriptor,
-    only: [
-      get_param_description: 3,
-      get_content_type: 1,
-      get_attr_description: 2
-    ]
+  @doc """
+  Return an Info Object builded from the api_info suplied by the `Xcribe.Information`.
+  """
+  def info_object(api_info) do
+    %{title: api_info.name, description: api_info.description, version: "1"}
```

Comment:
Maybe this is a matter for another PR, but why are we making this `version` hardcoded to `1`?

Reply:
Because the information about the API version should be give by `Information` module but we don't have this feature yet. I pretend improve our information module next PR. That and other features are required for a better documentation with swagger spec.

Reply:
If the question was about the "1" itself, really could be any thing "v1", "1.0" etc